appOPHD/UI/MiniMap.cpp

```diff
@@ -79,18 +79,18 @@ void MiniMap::draw() const
 		}
 	}
 
-	for (auto minePosition : mTileMap.oreDepositLocations())
+	for (auto oreDepositPosition : mTileMap.oreDepositLocations())
 	{
-		OreDeposit* mine = mTileMap.getTile({minePosition, 0}).oreDeposit();
-		if (!mine) { break; } // avoids potential race condition where a mine is destroyed during an updated cycle.
+		OreDeposit* oreDeposit = mTileMap.getTile({oreDepositPosition, 0}).oreDeposit();
+		if (!oreDeposit) { break; } // avoids potential race condition where an Ore Deposit is destroyed during an updated cycle.
 
 		auto mineBeaconStatusOffsetX = 0;
-		if (!mine->active()) { mineBeaconStatusOffsetX = 0; }
-		else if (!mine->exhausted()) { mineBeaconStatusOffsetX = 8; }
+		if (!oreDeposit->active()) { mineBeaconStatusOffsetX = 0; }
+		else if (!oreDeposit->exhausted()) { mineBeaconStatusOffsetX = 8; }
```

Comment:
On another unrelated matter, after the name change this part really stands out. What does it mean for an `oreDeposit` to be `active`? That sounds really weird.